ee/ucp/kubernetes/install-cni-plugin.md

```diff
@@ -1,98 +1,117 @@
-  ```
-
-## Disable IP in IP overlay tunneling
-
-The Calico CNI plugin supports both overlay (IPIP) and underlay forwarding
-technologies. By default, Docker UCP uses IPIP overlay tunneling.
-
-If you're used to managing applications at the network level through the 
-underlay visibility, or you want to reuse existing networking tools in the
-underlay, you may want to disable the IPIP functionality. Run the following
-commands on the Kubernetes master node to disable IPIP overlay tunneling.
+Once the installation is complete, you will be able to access UCP in web
+browser. Note the Manager node will be unhealthy as the Kubelet will be
+reporting `NetworkPluginNotReady`. Additionally the metrics in the UCP dashboard
+will also be unavailable, as this runs in a Kubernetes pods.
+
+### Configure CLI Access to UCP
+
+Next a platform operator should log into UCP, download a UCP Client Bundle, and
+configure the Kubernetes CLI tool `kubectl`. For more information see [CLI Based
+Access](ee/ucp/user-access/cli/#download-client-certificates)
+   
+Using `kubectl` you should be able to see that the UCP components that run on
+Kubernetes are still pending, waiting for a CNI driver before becoming
+available. 
 
 ```bash
-# Exec into the Calico Kubernetes controller container.
-docker exec -it $(docker ps --filter name=k8s_calico-kube-controllers_calico-kube-controllers -q) sh
+$ kubectl get nodes
+NAME         STATUS     ROLES     AGE       VERSION
+manager-01   NotReady   master    10m       v1.11.9-docker-1
+  
+$ kubectl get pods -n kube-system -o wide
+NAME                           READY     STATUS              RESTARTS   AGE       IP        NODE         NOMINATED NODE
+compose-565f7cf9ff-gq2gv       0/1       Pending             0          10m       <none>    <none>       <none>
+compose-api-574d64f46f-r4c5g   0/1       Pending             0          10m       <none>    <none>       <none>
+kube-dns-6d96c4d9c6-8jzv7      0/3       Pending             0          10m       <none>    <none>       <none>
+ucp-metrics-nwt2z              0/3       ContainerCreating   0          10m       <none>    manager-01   <none>
+```
+
+### Install an Unmanaged CNI Plugin
 
-# Download calicoctl
-wget https://github.com/projectcalico/calicoctl/releases/download/v3.1.1/calicoctl && chmod +x calicoctl
+Using the `kubectl` tool you are now able to install a custom CNI plugin on to
+the Universal Control Plane. Alternative CNI plugins could be Weave, Flannel,
+Canal, Romana and many more. As Docker Support will not support the CNI plugin,
+a platform operator has complete flexibility on what they install.
 
-# Get the IP pool configuration. 
-./calicoctl get ippool -o yaml > ippool.yaml
+Common steps when installing a CNI plugin would be to download the relevant
+upstream CNI binaries from
+https://github.com/containernetworking/cni/releases/tag/, placing them in
+`/opt/cni/bin`, additionally downloading the relevant CNI Plugins Kubernetes
```

Comment:
can we mention file names?
I know for Cisco ACI/CNI, it's loopback. 
@nerdalert

Reply:
I've left this out of the Doc as it various so much on which CNI Plugin you use, even to the point which version you need 0.3.0 or 0.6.0 etc :)